lighttpd/tests/docker/auth/docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: '3.5'
 
 services:
   lighttpd:
-    image: sebp/lighttpd
+    image: sebp/lighttpd:${LIGHTTPD_VERSION}-r0
```

Comment:
Hmm, not sure about that `-r0` suffix…

*Alternatively* again… The [SSH integration](https://github.com/DataDog/integrations-core/blob/1445e1febd05f2eddf435870bbfc7b265650d14d/ssh_check/tox.ini) separates the environment variables for the version under test (used as an input for version metadata tests), and the actual image tag in use. Sounds like the *cleanest* approach to me:

https://github.com/DataDog/integrations-core/blob/1445e1febd05f2eddf435870bbfc7b265650d14d/ssh_check/tox.ini#L14-L15

So, here…

```ini
[testenv]
setenv =
    LIGHTTPD_VERSION = 1.4.54
    LIGHTTPD_IMAGE = sepb/lighttpd:1.4.54-r0
```

Reply:
Updated!